Android/app/src/main/res/menu/activity_main_drawer.xml

```diff
@@ -1,7 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <menu xmlns:android="http://schemas.android.com/apk/res/android">
 
-    <group android:checkableBehavior="single" android:id="@+id/nav_grp1">
+    <group
```

Comment:
these changes shouldnt be a part of your commit

Reply:
I don't know why this file was changed in the commit when I didn't touch it.